vcpkg.json

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "f3d",
+    "dependencies": [
+        "alembic",
+        "assimp",
+        "draco",
+        "opencascade",
+        {
+            "name": "vtk",
+            "default-features": false,
+            "features": [ "opengl" ]
+        }
+    ],
+    "builtin-baseline": "60e2c07d20aeb4bf534a8deabf92a70cbcb07617"
```

Comment:
no raytracing in vcpkg ?

Reply:
Looking at the port file of vtk, it does not seem so.